### ISSUE TITLE
A lot of layering and nesting in the Variability Screen studio tree.

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#388]
There has been a lot of discussion about the nesting of the tandems in the Variability screen which makes the studio tree very messy and convoluted.  Using this issue to start investigation and discussion on that issue.

**Comment by marlitas** [comment]
I had the idea that we don't need to expose the ToggleNode implementation in the tree, and that those don't need to be instrumented either way, so I uninstrumented the ToggleNodes and grouped their contents through a tandem name that was a bit easier to identify. I think this helped a lot actually. I only have it in this patch, but I'd be curious about what @catherinecarter and @arouinfar think, and if we should go this way for all the toggleNodes.

Here's some pictures as examples:
<img width="352" alt="image" src="https://github.com/phetsims/center-and-variability/assets/42476338/2db88c17-0680-49b2-bc41-af179c17bc2d">
<img width="340" alt="image" src="https://github.com/phetsims/center-and-variability/assets/42476338/15b3d76e-0c2e-448f-a9ad-970dfad808f2">

We can talk about this more during a meeting as well.

<details>

```diff
Subject: [PATCH] uninstrument toggle nodes
---
Index: js/variability/view/VariabilityPlotNode.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/variability/view/VariabilityPlotNode.ts b/js/variability/view/VariabilityPlotNode.ts
--- a/js/variability/view/VariabilityPlotNode.ts	(revision e623a9ff6c9ce1368e39f7e42ea570955d8ad645)
+++ b/js/variability/view/VariabilityPlotNode.ts	(date 1689973595107)
@@ -32,30 +32,27 @@
   public constructor( model: VariabilityModel, sceneModel: VariabilitySceneModel, playAreaNumberLineNode: NumberLineNode, providedOptions: CAVPlotOptions ) {
     super( providedOptions );
 
+    const scenePlotNodesTandem = providedOptions.tandem;
     // We need to specify CAVPlotNode manually because otherwise TypeScript will infer all Nodes as the first element (RangeNode), see https://github.com/phetsims/sun/issues/846
     const toggleNode = new ToggleNode<VariabilityMeasure, CAVPlotNode>( model.selectedVariabilityMeasureProperty, [ {
-      createNode: tandem => new RangeNode( model, sceneModel, playAreaNumberLineNode, {
+      createNode: () => new RangeNode( model, sceneModel, playAreaNumberLineNode, {
         parentContext: 'accordion',
-        tandem: tandem.createTandem( 'rangeNode' )
+        tandem: scenePlotNodesTandem.createTandem( 'rangeNode' )
       } ),
-      tandemName: 'rangeNode',
       value: VariabilityMeasure.RANGE
     }, {
-      createNode: tandem => new IQRNode( model, sceneModel, playAreaNumberLineNode, {
+      createNode: () => new IQRNode( model, sceneModel, playAreaNumberLineNode, {
         parentContext: 'accordion',
-        tandem: tandem.createTandem( 'iqrNode' )
+        tandem: scenePlotNodesTandem.createTandem( 'iqrNode' )
       } ),
-      tandemName: 'iqrNode',
       value: VariabilityMeasure.IQR
     }, {
-      createNode: tandem => new MADNode( model, sceneModel, playAreaNumberLineNode, {
+      createNode: () => new MADNode( model, sceneModel, playAreaNumberLineNode, {
         parentContext: 'accordion',
-        tandem: tandem.createTandem( 'madNode' )
+        tandem: scenePlotNodesTandem.createTandem( 'madNode' )
       } ),
-      tandemName: 'madNode',
       value: VariabilityMeasure.MAD
     } ], {
-      tandem: providedOptions.tandem.createTandem( 'toggleNode' ),
       alignChildren: ToggleNode.NONE
     } );
     this.addChild( toggleNode );
@@ -68,6 +65,8 @@
       new BooleanProperty( false ), {
         focusable: false,
         visibleProperty: model.isIntervalToolVisibleProperty,
+
+        // TODO: Does this need to be instrumented?
         tandem: providedOptions.tandem.createTandem( 'intervalToolNode' )
       } );
 
Index: js/variability/view/VariabilityAccordionBox.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/variability/view/VariabilityAccordionBox.ts b/js/variability/view/VariabilityAccordionBox.ts
--- a/js/variability/view/VariabilityAccordionBox.ts	(revision e623a9ff6c9ce1368e39f7e42ea570955d8ad645)
+++ b/js/variability/view/VariabilityAccordionBox.ts	(date 1689973561314)
@@ -48,21 +48,19 @@
 
     const accordionBoxTitleProperty = new DynamicProperty<string, unknown, unknown>( currentProperty );
 
+    const plotNodesTandem = tandem.createTandem( 'scenePlotNodes' );
     const contents = _.range( 4 ).map( i => {
       return {
         value: model.sceneModels[ i ],
-        createNode: ( tandem: Tandem ) => new VariabilityPlotNode( model, model.variabilitySceneModels[ i ], playAreaNumberLineNode, {
-          tandem: tandem,
+        createNode: () => new VariabilityPlotNode( model, model.variabilitySceneModels[ i ], playAreaNumberLineNode, {
+          tandem: plotNodesTandem.createTandem( `scene${i + 1}PlotNodes` ),
           bottom: backgroundShape.bounds.height
-        } ),
-
-        tandemName: 'plotNode' + ( i + 1 )
+        } )
       };
     } );
 
     const plotToggleNode = new ToggleNode( model.selectedSceneModelProperty, contents, {
-      bottom: backgroundShape.bounds.height,
-      tandem: tandem.createTandem( 'plotToggleNode' )
+      bottom: backgroundShape.bounds.height
     } );
 
     const infoButton = new CAVInfoButton( model.isInfoVisibleProperty, backgroundShape, tandem.createTandem( 'infoButton' ) );
@@ -72,24 +70,24 @@
     const textGroup = new AlignGroup();
     const iconGroup = new AlignGroup();
 
+    const checkboxesTandem = tandem.createTandem( 'checkboxes' );
     const checkboxToggleNode = new AlignBox( new ToggleNode( model.selectedVariabilityMeasureProperty, [ {
-      createNode: tandem => new VariabilityMeasureCheckbox( model.isRangeVisibleProperty,
-        CenterAndVariabilityStrings.rangeStringProperty, iconGroup, textGroup, CAVColors.rangeFillProperty, { tandem: tandem } ),
-      tandemName: 'rangeAccordionCheckbox',
+      createNode: () => new VariabilityMeasureCheckbox( model.isRangeVisibleProperty,
+        CenterAndVariabilityStrings.rangeStringProperty, iconGroup, textGroup, CAVColors.rangeFillProperty,
+        { tandem: checkboxesTandem.createTandem( 'rangeAccordionCheckbox' ) } ),
       value: VariabilityMeasure.RANGE
     }, {
-      createNode: tandem => new VariabilityMeasureCheckbox( model.isIQRVisibleProperty,
-        CenterAndVariabilityStrings.iqrStringProperty, iconGroup, textGroup, CAVColors.iqrColorProperty, { tandem: tandem } ),
-      tandemName: 'iqrAccordionCheckbox',
+      createNode: () => new VariabilityMeasureCheckbox( model.isIQRVisibleProperty,
+        CenterAndVariabilityStrings.iqrStringProperty, iconGroup, textGroup, CAVColors.iqrColorProperty,
+        { tandem: checkboxesTandem.createTandem( 'iqrAccordionCheckbox' ) } ),
       value: VariabilityMeasure.IQR
     }, {
       createNode: tandem => new VariabilityMeasureCheckbox( model.isMADVisibleProperty,
-        CenterAndVariabilityStrings.madStringProperty, iconGroup, textGroup, CAVColors.madRectangleColorProperty, { tandem: tandem } ),
-      tandemName: 'madAccordionCheckbox',
+        CenterAndVariabilityStrings.madStringProperty, iconGroup, textGroup, CAVColors.madRectangleColorProperty,
+        { tandem: checkboxesTandem.createTandem( 'madAccordionCheckbox' ) } ),
       value: VariabilityMeasure.MAD
     } ], {
-      alignChildren: ToggleNode.RIGHT,
-      tandem: tandem.createTandem( 'checkboxesToggleNode' )
+      alignChildren: ToggleNode.RIGHT
     } ), { alignBounds: backgroundShape.bounds, xAlign: 'right', yAlign: 'center' } );
 
     backgroundNode.addChild( plotToggleNode );
@@ -139,15 +137,14 @@
       } );
     };
 
-
+    const readoutsTandem = tandem.createTandem( 'valueReadouts' );
     const readoutsToggleNode = new AlignBox( new ToggleNode( model.selectedVariabilityMeasureProperty, [ {
       value: VariabilityMeasure.RANGE,
-      tandemName: 'rangeReadoutToggleNode',
-      createNode: tandem => {
+      createNode: () => {
         const rangeEqualsValueStringProperty = deriveStringProperty( vsm => vsm.rangeValueProperty,
           CenterAndVariabilityStrings.rangeUnknownValueStringProperty, rangePatternStringProperty );
 
-        const rangeReadoutTextTandem = tandem.createTandem( 'rangeReadoutText' );
+        const rangeReadoutTextTandem = readoutsTandem.createTandem( 'rangeReadoutText' );
         const rangeReadoutText = new VariabilityReadoutText( rangeEqualsValueStringProperty, {
           fill: CAVColors.meanColorProperty,
           visibleProperty: createGatedVisibleProperty( model.isRangeVisibleProperty, rangeReadoutTextTandem ),
@@ -163,22 +160,21 @@
       }
     }, {
       value: VariabilityMeasure.IQR,
-      tandemName: 'iqrReadoutToggleNode',
-      createNode: tandem => {
+      createNode: () => {
 
         const medianEqualsValueStringProperty = deriveStringProperty( vsm => vsm.medianValueProperty,
           CenterAndVariabilityStrings.medianUnknownValueStringProperty, medianPatternStringProperty );
 
         const medianReadoutText = new VariabilityReadoutText( medianEqualsValueStringProperty, {
           fill: CAVColors.medianColorProperty,
-          tandem: tandem.createTandem( 'medianReadoutText' ),
+          tandem: readoutsTandem.createTandem( 'medianReadoutText' ),
           phetioVisiblePropertyInstrumented: true
         } );
 
         const iqrEqualsValueStringProperty = deriveStringProperty( vsm => vsm.iqrValueProperty,
           CenterAndVariabilityStrings.iqrUnknownValueStringProperty, iqrPatternStringProperty );
 
-        const iqrReadoutTextTandem = tandem.createTandem( 'iqrReadoutText' );
+        const iqrReadoutTextTandem = readoutsTandem.createTandem( 'iqrReadoutText' );
         const iqrReadoutText = new VariabilityReadoutText( iqrEqualsValueStringProperty, {
           fill: CAVColors.iqrLabelColorProperty,
           visibleProperty: createGatedVisibleProperty( model.isIQRVisibleProperty, iqrReadoutTextTandem ),
@@ -198,8 +194,7 @@
       }
     }, {
       value: VariabilityMeasure.MAD,
-      tandemName: 'madReadoutToggleNode',
-      createNode: tandem => {
+      createNode: () => {
 
         const meanEqualsValueStringProperty = deriveStringProperty( vsm => vsm.meanValueProperty,
           CenterAndVariabilityStrings.meanUnknownValueStringProperty, meanPatternStringProperty );
@@ -209,11 +204,11 @@
 
         const meanReadoutText = new VariabilityReadoutText( meanEqualsValueStringProperty, {
           fill: CAVColors.meanColorProperty,
-          tandem: tandem.createTandem( 'meanReadoutText' ),
+          tandem: readoutsTandem.createTandem( 'meanReadoutText' ),
           phetioVisiblePropertyInstrumented: true
         } );
 
-        const madReadoutTextTandem = tandem.createTandem( 'madReadoutText' );
+        const madReadoutTextTandem = readoutsTandem.createTandem( 'madReadoutText' );
         const madReadoutText = new VariabilityReadoutText( madEqualsValueStringProperty, {
           fill: CAVColors.madColorProperty,
           visibleProperty: createGatedVisibleProperty( model.isMADVisibleProperty, madReadoutTextTandem ),
@@ -232,8 +227,7 @@
         } );
       }
     } ], {
-      alignChildren: ToggleNode.NONE,
-      tandem: tandem.createTandem( 'readoutsToggleNode' )
+      alignChildren: ToggleNode.NONE
     } ), { alignBounds: backgroundShape.bounds, xAlign: 'left', yAlign: 'center' } );
 
     backgroundNode.addChild( readoutsToggleNode );
```

</details>

**Comment by catherinecarter** [comment]
There's a possible reorganization in #418 that will likely address this. Feel free to close this issue as it will most likely be addressed in the other issue, as well as #394, and #395.

**Comment by marlitas** [comment]
Sounds good. Closing as recommended above.
